README.md

```diff
@@ -16,31 +16,100 @@ freeCodeCamp is a friendly open source community where you learn to code and hel
+
+
+###These six new certifications will replace the old legacy certificates:
+
+**Legacy Front End Development**
+**Legacy Data Visualization**
+**Legacy Back End Development**
+
+###Coding for nonprofits
+
+We’ve decided to make coding for nonprofits a separate process from the certificate program. We’ve added 5 of our projects that are used by nonprofits - our Open Source for Good projects - to the challenge map. Feel free to browse their open issues. You can dive in and start contributing whenever you feel ready.
+
+We will continue to add Open Source for Good projects to the challenge map as they become stable and ready for your contributions. These projects are all built using the same JavaScript-based tools you’ll learn by earning the certificates. And several of these are nearing a public beta (I’ll formally announce these as they go into open beta).
+
+In addition to our Open Source for Good projects - which are particularly well-documented and beginner-friendly - there are tons of other open source projects you can contribute to. The community has built a guide to contributing to various open source projects here 363.
+
+###What happens to the old legacy certificates?
+
+If you’ve already earned a certificate, you will always have it. You will always be able to link to it from your LinkedIn or résumé. And when your prospective employers or freelance clients click that link, they’ll see a verified certificate specific to you.
+
+And if you’ve already started working toward the Legacy Front End Development, Legacy Data Visualization, or Legacy Back End Development certificates, and still want to claim it, don’t worry - you can will always be able to claim these certificates - even after we release the beta challenges.
+
+All of the required projects for these old certificates will remain available on freeCodeCamp. For the old certificates, you can complete old versions of the projects.
+
+As long as you created your account before we release the expanded curriculum, you will be able to claim these old certificates any time you want, and they’ll start showing up on your freeCodeCamp profile.
+
+Campers who create their freeCodeCamp after we release the expanded curriculum will not be able to claim these old certificates.
+
+Note that in order to claim the new certificates, you’ll need to build the new projects (some of which are updated versions of old projects) and get all of their tests passing.
+
+###Some cool facts about the new certificates
+
+1. Each of these certificates has 5 projects, all of which have built-in test suites. This means that each user story has a corresponding test that can tell you when you’ve properly fulfilled it.
+2. You can pull in these tests through a CDN, meaning you’ll be able to build these projects on websites like CodePen, or your local development environment, or even on freeCodeCamp.org 80 itself.
```

Comment:
You can pull in these tests through a CDN, meaning you’ll be able to build these projects on websites like CodePen, or your local development environment ~, or even on freeCodeCamp.org 80 itself~.

This is not possible yet.